doc/rest-api/Metrics-backend.md

```diff
@@ -35,24 +35,25 @@ The above configuration starts the API only on a loopback interface.
 The responses are composed in a JSON format with a root element containing one or more attributes as response elements.
 
 Example response:
-
-    {
-        "hosts": [
-            "localhost"
-        ],
-        "metrics": [
-            "xmppErrorIq",
-            "xmppPresenceReceived",
-            "xmppMessageBounced",
-            (...)
-        ],
-        "global": [
-            "nodeSessionCount",
-            "totalSessionCount",
-            "uniqueSessionCount",
-            (...)
-        ]
-    }
+```json
+{
+    "hosts": [
+        "localhost"
+    ],
+    "metrics": [
+        "xmppErrorIq",
+        "xmppPresenceReceived",
+        "xmppMessageBounced",
+        (...)
+    ],
+    "global": [
+        "nodeSessionCount",
+        "totalSessionCount",
+        "uniqueSessionCount",
+        (...)
```

Comment:
Same as above

Reply:
will do

Reply:
well basing on [this](https://stackoverflow.com/questions/244777/can-comments-be-used-in-json) discussion I doubt we can actually have it commented 😞